app/Http/Controllers/LandingPagesController.php

```diff
@@ -0,0 +1,258 @@
+            $mainFeatures = $item->homeFeatures()->published()->limit(3)->get();
+        }
+
+        // Visit
+        $video_url = $item->file('video', 'en');
+
+        if ($video_url) {
+            $headerImage = $item->imageFront('visit_mobile');
+
+            $poster_url = $headerImage['src'] ?? '';
+            $video = [
+                'src' => $video_url,
+                'poster' => $poster_url,
+                'fallbackImage' => $headerImage,
+            ];
+
+            $headerMedia = [
+                'type' => 'video',
+                'size' => 'hero',
+                'media' => $video,
+                'hideCaption' => true,
+            ];
+        } else {
+            $headerMedia = [
+                'type' => 'image',
+                'size' => 'hero',
+                'media' => $item->imageFront('visit_hero'),
+                'hideCaption' => true,
+            ];
+        }
+
+        $hours = [
+            'hide_hours' => $item->visit_hide_hours,
+            'media' => [
+                'type' => 'image',
+                'size' => 's',
+                'media' => $item->imageFront('visit_featured_hour'),
+                'caption' => $item->visit_hour_image_caption,
+            ],
+            'primary' => $item->visit_hour_header,
+            'secondary' => $item->visit_hour_subheader,
+            'sections' => $item->featured_hours,
+            'intro' => $item->visit_hour_intro
+        ];
+
+        $itemprops = [
+            'name' => 'Art Institute of Chicago',
+            'telephone' => '+13124433600',
+            'publicAccess' => 'true',
+        ];
+
+        $contrastHeader = false;
+        $filledLogo = false;
+        $title = '';
+        if ($item->type === LandingPage::PAGE_TYPE_HOME) {
+            $this->seo->setTitle("Downtown Chicago's #1 Museum");
+            $this->seo->setDescription("Located downtown by Millennium Park, this top art museum is TripAdvisor's #1 Chicago attraction—a must when visiting the city.");
+            $contrastHeader = sizeof($mainFeatures) > 0;
+            $filledLogo = sizeof($mainFeatures) > 0;
+        } elseif ($item->type === LandingPage::PAGE_TYPE_VISIT) {
+            $this->seo->setTitle('Visit a Chicago Landmark');
+            $this->seo->setDescription('Looking for things to do in Downtown Chicago? Plan your visit, find admission pricing, hours, directions, parking & more!');
+            $this->seo->setImage($item->imageFront('visit_hero') ?? $item->imageFront('visit_mobile'));
+            $contrastHeader = true;
+            $filledLogo = true;
+            $title = __('Visit');
+        } elseif ($item->type === LandingPage::PAGE_TYPE_RESEARCH_LANDING) {
+            $this->seo->setTitle('Research & Resources');
+            $this->seo->setDescription($item->resources_landing_intro);
+            $this->seo->setImage($item->imageFront('research_landing_image'));
+            $title = 'The Collection';
+        }
```

Comment:
For this type of conditional, where it's always comparing against one variable (`$item->type`), I like to use switch statements instead.